packages/aws/src/clients/cloudformation.client.ts

```diff
@@ -80,6 +80,21 @@ export class CloudformationClient implements CloudformationClientInterface {
         }
     }
 
+    /**
+     * Gets the description and all its details from all the CloudFormation stacks.
+     */
+    async listStacks(): Promise<Stack[] | undefined> {
```

Comment:
Can this command really return undefined or just an empty array ?

Reply:
Apparently undefined but you are right I prefer an empty array.